.travis.yml

```diff
@@ -6,3 +6,8 @@ perl:
     - "5.18"
     - "5.16"
     - "5.14"
+before_install:
+    - eval $(curl https://travis-perl.github.io/init) --auto
+    - local-lib
+    - git clone --depth=1 --branch=develop https://github.com/zonemaster/zonemaster-engine.git
+    - cpan-install --deps ./zonemaster-engine
```

Comment:
I have no objections but one question. Wouldn't downloading the zip file with the checkout files in the develop branch giving less overhead to Travis and possibly being faster? The Git structure is of no importance here.

https://github.com/zonemaster/zonemaster-engine/archive/develop.zip